test/extensions/filters/http/dynamo/dynamo_request_parser_test.cc

```diff
@@ -74,6 +74,72 @@ TEST(DynamoRequestParser, parseTableNameSingleOperation) {
   }
 }
 
+TEST(DynamoRequestParser, parseTableNameTransactOperation) {
+  std::vector<std::string> supported_transact_operations{"TransactGetItems", "TransactWriteItems"};
```

Comment:
You could use the statically defined constant vector `RequestParser::TRANSACT_OPERATIONS` to prevent drift, no?

Reply:
true!

Reply:
so this list is private, meaning i can't call it outside of the class. and all the other operations vectors are private as well, meaning that i'd either have to convert them all to public or not follow the set convention, so i'm just reverting back to having a separate list in the test (which is what the other tests for this file do)